internal/jobservice/application.go

```diff
@@ -41,19 +42,26 @@ func (a *App) StartUp(ctx context.Context) error {
 	inMemoryMap := make(map[string]*repository.JobTable)
 	subscribedJobSets := make(map[string]*repository.SubscribeTable)
 	jobStatusMap := repository.NewJobStatus(inMemoryMap, subscribedJobSets)
-	inMemoryJobService := repository.NewInMemoryJobServiceRepository(jobStatusMap, config)
-	jobService := server.NewJobService(config, *inMemoryJobService)
+
+	db, err := sql.Open("sqlite", "jobservice.db")
+	if err != nil {
+		log.Errorf("Error Opening Sqlite DB from jobservice.db %v", err)
```

Comment:
I think you can do `log.withError(err).Err` (at least you can with logrus) which gives you the stacktrace.

Reply:
also if you have an error here I think you might want to exit?  (or possibly try and instantiate a nnew db).  I don't think trying to run the below code with a broken db will end well!